policyengine_us/variables/gov/states/nm/tax/income/nm_subtractions.py

```diff
@@ -0,0 +1,19 @@
+from policyengine_us.model_api import *
+
+
+class nm_subtractions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "New Mexico income subtractions"
+    unit = USD
+    definition_period = YEAR
+    defined_for = StateCode.NM
+
+    adds = [
+        "tax_exempt_interest_income",  # Line 6
+        "dividend_income",  # Line 6
+        "us_govt_interest",  # Line 8
+        # "tax_unit_taxable_social_security",  # includes railroad retirement benefits, Line 9, Line 24
```

Comment:
Why commented?

Reply:
Part of tax_unit_taxable_social_security is included nm_social_security_income_exemption

Reply:
So is the remainder subtracted?